plugins/repository-s3/build.gradle

```diff
@@ -169,24 +169,17 @@ if (useFixture) {
 
   File minioAddressFile = new File(project.buildDir, 'generated-resources/s3Fixture.address')
 
-  // We can't lazy evaluate a system property for the Minio address passed to JUnit so we write it to a resource file
-  // and pass its name instead.
-  task writeMinioAddress {
+  thirdPartyTest {
     dependsOn tasks.bundlePlugin, tasks.postProcessFixture
     outputs.file(minioAddressFile)
-    doLast {
+    doFirst {
```

Comment:
This breaks because we are declaring `miniAddressFile` as both an output **and** input of this task. What's wrong with keeping this as a separate task? Either that or remove the input declaration.

Reply:
I removed the input declaration, it's really just an output now. 
We don't need the file at all ( as per the `TODO` ) so not having a task to write it is a step in that direction.